crates/bevy_gltf/src/lib.rs

```diff
@@ -49,7 +49,9 @@ impl GltfPlugin {
 impl Plugin for GltfPlugin {
     fn build(&self, app: &mut App) {
         app.register_type::<GltfExtras>()
+            .register_type::<GltfExtensions>()
             .init_asset::<Gltf>()
+            .register_type::<WeakGltf>()
```

Comment:
Small Nitpick, but I think this order looks better 
```suggestion
            .register_type::<GltfExtensions>()
            .register_type::<WeakGltf>()
            .init_asset::<Gltf>()
```